test/build.bats

```diff
@@ -355,6 +367,7 @@ apply -p1 -i /my/patch.diff
 exec ./configure "\$@"
 CONF
 
+  stub uname '-s : echo Linux'
```

Comment:
When `uname -s` returns `FreeBSD`,
ruby-build adds configure option '--with-readline-dir=/usr/local'.
It is a result of my previous Pull Request.

Some error message are shown like below.

```
✗ yaml is installed for ruby
   (from function `assert_equal' in file test/test_helper.bash, line 139,
    from function `assert_output' in file test/test_helper.bash, line 148,
    from function `assert_build_log' in file test/build.bats, line 58,
    in test file test/build.bats, line 76)
     `assert_build_log <<OUT' failed
   expected: yaml-0.1.6: --prefix=${TMP}/install
   make -j 2
   make install
   ruby-2.0.0: --prefix=${TMP}/install
   make -j 2
   make install
   actual:   yaml-0.1.6: --prefix=${TMP}/install
   make -j 2
   make install
   ruby-2.0.0: --prefix=${TMP}/install --with-readline-dir=/usr/local
   make -j 2
   make install
```